packages/editor/src/components/DataSource/ApiForm/Basic.tsx

```diff
@@ -145,6 +147,32 @@ export const Basic: React.FC<Props> = props => {
           onBlur={() => formik.handleSubmit()}
         />
       </FormControl>
+
+      <FormControl display="flex" alignItems="center">
+        <FormLabel margin="0" marginRight="2">
+          Disabled
+        </FormLabel>
+        <SpecWidget
+          component={api}
+          spec={Type.Boolean({ widgetOptions: { isShowAsideExpressionButton: true } })}
+          value={formik.values.disabled}
+          path={[]}
+          level={1}
+          services={services}
+          onChange={val => {
```

Comment:
The API form component's render is heavy, should use `useCallback` and `useMemo` to cache the `spec`, `path`, and `onChange` props to avoid unnecessary render.